statgen/pheget
# Show DAP-G on Y-axis

## Ticket
n/a

## Purpose
Add an additional field (Posterior Inclusion Probability - PIP, results from Wen et al.'s DAP-G analysis of GTEx v8) to be plotted on the Y-axis (where the data is available). This requires a new database file containing the additional data.

Since the additional data is somewhat sparse, my approach is to retrieve the data for a given query, convert this into a Python dictionary, pass that dictionary to the parser, and add the PIP values to the points if possible. This approach should broadly work well enough for any additional data we may want to add.

## How to test this
View the single-variant view for the test variant chr19:488506, and click the "PIP" button to ensure the correct variable is plotted on the Y-axis. Switch back and forth between other Y-axis variables, and other buttons, to confirm that it works (and that I didn't break something else). 

## Deployment / configuration notes
### Adds new data
Download the test database file (data/GTEx_v8_finemapping_DAPG.sqlite.db) to your current data directory (as specified in your local settings) - please note that this test file contains only information for chr19:488506. For all available data, you can use the full database (~3GB), which you can find at the following location: /net/amd/amkwong/browseQTL/data/GTEx_v8_finemapping_DAPG.sqlite.db

### Modifies dependencies
Be sure to update python dependencies (per requirements.txt) , to reflect the new required version of zorp.
